crates/api_models/src/admin.rs

```diff
@@ -240,6 +251,13 @@ pub enum RoutingAlgorithm {
     Single(api_enums::RoutableConnectors),
 }
 
+#[derive(Clone, Debug, Deserialize, ToSchema, Serialize)]
+#[serde(deny_unknown_fields)]
+pub struct PrimaryBusinessDetails {
+    country: String,
```

Comment:
create a country enum. Can we taken as separate PR.

Reply:
we are using country as String, needs to be changed at all places ?

Reply:
yes